frontend/src/components/utils/ProgressBar.tsx

```diff
@@ -1,25 +1,53 @@
 import React from "react";
-import { Progress } from "@chakra-ui/react";
+import { Progress, Text } from "@chakra-ui/react";
 import "./ProgressBar.css";
 import { truncate } from "../../utils/Utils";
 
 type ProgressBarProps = {
   percentageComplete: number;
   type: string;
+  fontSize: string;
 };
 
-const ProgressBar = ({ percentageComplete, type }: ProgressBarProps) => {
+const ProgressBar = ({
+  percentageComplete,
+  type,
+  fontSize,
+}: ProgressBarProps) => {
+  let progressFontSize;
+  let progressWidth;
+  let progressMargin;
+  switch (fontSize) {
+    case "12px":
+      progressFontSize = "10px";
+      progressWidth = "130px";
+      progressMargin = "10px";
+      break;
+    case "16px":
+      progressFontSize = "12px";
+      progressWidth = "155px";
+      progressMargin = "15px";
+      break;
+    case "24px":
+      progressFontSize = "16px";
+      progressWidth = "210px";
+      progressMargin = "20px";
+      break;
+    default:
+      break;
+  }
+
   return (
     <div className="progress-bar-container">
```

Comment:
do you mind converting this file to chakraui? ie, for this div
`<Box maxWidth="230px">`
And then swap the div on line 45 with a Box too

Reply:
after this change, it seems that the ProgressBar.css file can be removed too! hurray!